README.md

```diff
@@ -7,6 +7,7 @@ All of your solutions in one place - use this to help you plan for FAC11
 [Accessibility](https://github.com/njsfield/accessibility-challenge)
 + [Your names](nice-link-here)
 + [Your names](nice-link-here)
```

Comment:
@oliverjam This is great :grin: 

Please can you remove these "Your names (nice-link-here)" lines? They was just intended as placeholder text to show you guys the format for inserting your own.